lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -12,6 +12,10 @@ class MarriageAbroadDataQuery
       france monaco new-caledonia wallis-and-futuna
     )
 
+    FRENCH_OVERSEAS_TERRITORIES_OFFERING_PACS = %w(
+      new-caledonia wallis-and-futuna
+    )
+
```

Comment:
Minor: As I mentioned earlier the formatting of arrays in this file seems inconsistent.